providers/tests/alibaba/cloud/log/test_oss_task_handler.py

```diff
@@ -22,14 +22,13 @@
 from unittest.mock import PropertyMock
 
 import pytest
+from tests_common.test_utils.config import conf_vars
+from tests_common.test_utils.db import clear_db_dags, clear_db_runs
```

Comment:
Please add this to the ruff config section in pyproject.toml so that it knows this is a "testing" group.

Reply:
Very good point. That will limit a number of changes! Thanks!

Reply:
Fixed. Will push it shortly.